routes/chat.js

```diff
@@ -6,16 +6,27 @@ const chatController = require('../controllers/chat/ChatController');
 const auth = require('../middlewares/auth');
 const isTargetUserAllowingAnonDMMiddleware = require('../middlewares/chat/isTargetUserAllowingAnonDMMiddleware');
 const SuccessMiddleware = require('../middlewares/success');
-const sendSignedMessage = require('../controllers/chat/sendSignedMessage');
-const setSignedChannelAsRead = require('../controllers/chat/setSignedChannelAsRead');
 const BodyValidationMiddleware = require('../middlewares/body-validation');
 const getSignedChannelList = require('../controllers/chat/getSignedChannelList');
+const {
+  sendFollowSystemMessage,
+  setSignedChannelAsRead,
+  sendSignedMessage
+} = require('../controllers/chat');
+const {ChatValidation} = require('../joi-validations/chat.validation');
+const {validate} = require('../middlewares/joi-validation/validate');
 
 router.get('/create-channel', chatController.createChannel);
 router.post('/add-moderator', chatController.addChannelModerator);
 router.post('/add-members-channel', auth.isAuth, chatController.addMembers);
 router.post('/anonymous', auth.isAuthAnonim, chatController.sendAnonymous);
 router.post('/send-signed-message', auth.isAuthV2, sendSignedMessage);
+router.post(
+  '/send-follow-system-message',
+  validate(ChatValidation.sendFollowSystemMessage),
+  auth.isAuthV2,
+  sendFollowSystemMessage
+);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `/send-follow-system-message` endpoint looks correct. The route uses the `validate` middleware with the `ChatValidation.sendFollowSystemMessage` schema, which should ensure that the request body is properly validated before reaching the controller. The `auth.isAuthV2` middleware is also used, which should ensure that only authenticated users can access this endpoint.
```diff
+ router.post(
+   '/send-follow-system-message',
+   validate(ChatValidation.sendFollowSystemMessage),
+   auth.isAuthV2,
+   sendFollowSystemMessage
+ );
```
Please ensure that the `ChatValidation.sendFollowSystemMessage` schema correctly validates all necessary fields in the request body.


<!-- This is an auto-generated comment by OSS CodeRabbit -->